### PR TITLE
Fix minor issues + Add tests

### PR DESCRIPTION
**Pull request by egeloen**
All are in the title...

ping @tmartin

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+before_script:
+    - composer require --no-update symfony/framework-bundle:${SYMFONY_VERSION}
+    - composer install --dev
+
+script: bin/phpunit
+
+notifications:
+    email:
+      - geloen.eric@gmail.com
```
Comment (egeloen) [inline .travis.yml]: @tmartin Would you like I add you?

Comment (tmartin) [inline .travis.yml]: Yep :)

Comment (egeloen) [inline .travis.yml]: Done

**Comment by egeloen** [comment]
The PR is finished. ping @tmartin

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
-The bundle allows to issue HTTP request.
+# README
 
-Currently, the adapters supported are:
+[![Build Status](https://secure.travis-ci.org/widop/WidopHttpAdapterBundle.png)](http://travis-ci.org/widop/WidopHttpAdapterBundle)
+
+The bundle allows to issue HTTP request. Currently, the adapters supported are:
```
Comment (tmartin) [inline README.md]: requestS

File: Resources/doc/installation.md [hunk 3]
```diff
@@ -1,89 +1,75 @@
 # Installation
 
-## Symfony 2.1.*
+## Symfony >= 2.1.*
 
-### Add the WidopHttpAdapterBundle to your composer configuration
+Require the bundle in your composer.json file:
```
Comment (tmartin) [inline Resources/doc/installation.md]: It light be helpful to add a text like this : _(choose the tag or branch that fits to your needs)._

File: Resources/doc/installation.md [hunk 4]
```diff
@@ -1,89 +1,75 @@
...
-### Add the WidopHttpAdapterBundle to your vendor/bundles/ directory
+```
+$ composer update
+```
 
-#### Using the vendors script
+## Symfony <= 2.0.*
 
-Add the following lines in your ``deps`` file
+Add Wid'op Http Adapter bundle:
```
Comment (tmartin) [inline Resources/doc/installation.md]: Same here.

File: README.md (hunk 2 above)
Comment (egeloen) [inline README.md]: Fixed

File: Resources/doc/installation.md (hunk 3 above)
Comment (egeloen) [inline Resources/doc/installation.md]: Fixed

File: Resources/doc/installation.md (hunk 4 above)
Comment (egeloen) [inline Resources/doc/installation.md]: Fixed

**Comment by tmartin** [comment]
Review :white_check_mark: 
Tests :white_check_mark: 

Waiting for Travis before merging.

**Comment by tmartin** [comment]
Travis build is OK. The failures are due to composer which could not fetch some dependencies.

Good to merge :+1:

**Pull request merged**